src/shared/_helpers/ArrayHelpers.ts

```diff
@@ -0,0 +1,33 @@
+class ArrayHelpers {
+  /**
+   * Get a random item in the array.
+   *
+   * @param {any[]} array Array
+   * @return {boolean}
+   */
+  static getRandomItem(array: any[]): any {
+    return array[Math.floor(Math.random() * array.length)];
+  }
+
+  /**
+   * Check if a value is an array.
+   *
+   * @param {any} value Value to check
+   * @return {boolean}
+   */
+  static isArray(value: any): any {
+    return Array.isArray(value);
+  }
+
```

Comment:
Can you fix the any types ? 
Here, something that should work 

```suggestion
   * @param {unknown} value Value to check
   * @return {boolean}
   */
  static isArray(value: unknown): boolean {
    return Array.isArray(value);
  }

```